judgels-backends/uriel/uriel-api/src/main/java/judgels/uriel/api/contest/role/ContestRole.java

```diff
@@ -0,0 +1,23 @@
+package judgels.uriel.api.contest.role;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum ContestRole {
+    @JsonProperty ADMIN("ADMIN"),
```

Comment:
We don't need a custom `@JsonProperty` if the spelling is exactly the same. Just use a normal enum; it's fine